app/minutes/controllers.py

```diff
@@ -31,10 +31,6 @@
 @get_user
 def get_minute(user, user_data):
     minute = Minutes.query.filter_by(id= user_data.get("minute_id", -1)).first()
-    
-    if user is None:
```

Comment:
`user is None` means that the user is not logged in. Since unauthenticated users should be allowed to view public data, this guard is removed.